### PR TITLE
Complete view-wedding implementation

### PR DESCRIPTION
**Pull request by ywllowsensor**
weddingKeyword is case-insensitive but has to match EXACTLY (spelling, spacing, format etc)

### CODE REVIEW

**Review by jonushzw — COMMENTED** [review]
changing from an array list to normal string solves a lot of problems! LGTM!

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -199,8 +199,6 @@ private CommandResult executeCommand(String commandText) throws CommandException
             if (commandText.startsWith(ListWeddingCommand.COMMAND_WORD)) {
                 weddingListPanel = new WeddingListPanel(logic.getFilteredWeddingList());
                 listPanelPlaceholder.getChildren().add(weddingListPanel.getRoot());
-            } else if (commandText.startsWith("view-wed")) {
-                // to fill
             } else {
```
Comment (yastsc) [inline src/main/java/seedu/address/ui/MainWindow.java:202]: Good job differentiating the personListPanel & weddingListPanel and putting it in the appropriate place!

**Review by yastsc — COMMENTED** [review]
LGTM!

File: src/main/java/seedu/address/model/person/TagContainsKeywordsPredicate.java [hunk 2]
```diff
@@ -24,15 +23,10 @@ public boolean test(Person person) {
...
-                        tagNames.stream()
-                                .anyMatch(tagName -> {
-                                    // Split the tag name into words and check if any word matches the keyword
-                                    String[] words = tagName.split("\\s+");
-                                    return Arrays.asList(words).contains(keyword.toLowerCase());
-                                })
-                );
+        boolean matchesWedding = tagNames.stream()
+                                    .anyMatch(tagName -> {
+                                        return tagName.toLowerCase().equals(weddingKeywords);
```
Comment (siriousguy) [inline src/main/java/seedu/address/model/person/TagContainsKeywordsPredicate.java:28]: Good solution to ensure exact match!

**Pull request merged**